src/css/style.css

```
@@ -1,28 +1,109 @@
-body{
-  background: lightslategray;
+body {
+  background: #eceef1;
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+}
+
+.logo-image {
+  margin: 10px 0 0 0;
+  height: fit-content;
+}
+
+footer {
+  text-align: center;
+  margin: auto;
+  padding: 40px;
+  font-size: 14px;
+  color: rgba(0, 0, 0, 0.7);
+  line-height: 22px;
+}
+
+h1 {
+  font-weight: 300;
+  color: rgba(0, 0, 0, 0.7);
+}
+
+em {
+  color: rgba(0, 0, 0, 0.7);
+  font-size: 14px;
+}
+
+a {
+  text-decoration: none;
+  color: #0077c5;
 }
 
 .div-top-bar {
-  background: aqua;
-  background-color: aqua;
+  width: calc(100vw - 60px);
+  text-align: right;
+  padding: 5px 15px;
+}
+
+.div-top-bar p {
+  margin-right: 20px;
 }
 
 .sign-up {
   margin: auto;
   width: 380px;
-  background: azure;
-  border-style: solid;
-  border-color: lightsteelblue;
-  border-width: 1px;
+  background: white;
+  border: 1px solid rgba(0, 0, 0, 0.2);
+  padding: 40px;
+  text-align: center;
 }
 
-.title-sign-up {
-  font-family: Arial, Helvetica, sans-serif;
+.form-element {
+  text-align: left;
+  margin: 40px 0;
+}
+
+.form-element > p {
+  margin: 5px 0;
+  font-size: 10px;
 }
 
-.i-email {
-  width: 320px;
-  margin-left: 30px;
-  margin-right: 30px;
+input {
+  border: 1px solid rgba(0, 0, 0, 0.2);
+  border-radius: 5px;
+  width: 100%;
+  height: 40px;
+}
 
-}
\ No newline at end of file
+.form-element input {
+  margin-top: 10px;
+}
+
+button {
+  width: 100%;
+  height: 50px;
+  background-color: #0077c5;
+  color: white;
+  font-size: 18px;
+  border: none;
+  border-radius: 5px;
+}
+
+.recaptcha-message {
+  color: rgba(0, 0, 0, 0.7);
+  background-color: rgba(0, 0, 0, 0.07);
+  width: calc(100% + 60px);
+  font-size: 12px;
+  padding: 10px;
+  margin: 20px -40px -40px -40px;
+}
+
+.sign-in {
+  width: 120px;
+  height: 40px;
+  display: inline-block;
+  vertical-align: middle;
+}
+
+.sign-in-text {
+  display: inline-block;
+  vertical-align: middle;
+  margin: 0;
+  color: rgba(0, 0, 0, 0.7);
+  line-height: 18px;
+  font-size: 14px;
```

Reduce the font size used, so your project looks as similar as possible to the clone. Like this part:
```suggestion
  font-size: 12px;
```